crates/rpc/rpc-api/src/eth.rs

```diff
@@ -143,6 +143,7 @@ pub trait EthApi {
         request: CallRequest,
         block_number: Option<BlockId>,
         state_overrides: Option<StateOverride>,
+        block_overrides: Option<BlockOverrides>,
```

Comment:
I would like to make this:
```suggestion
        block_overrides: Option<Box<BlockOverrides>>,
```
because this is not used very often